metadata-ingestion/src/datahub/ingestion/source/sql/sql_generic_profiler.py

```diff
@@ -267,14 +273,14 @@ def is_dataset_eligible_for_profiling(
                 self.config.profiling.profile_if_updated_since_days
             )
 
-        if not self.config.profile_pattern.allowed(dataset_name):
-            return False
-
```

Comment:
@treff7es did this get removed accidentally ?

Reply:
This line doesn't work because our profile pattern look like `db_name.schema_name.table_name.column_name` which never will match on table.

Reply:
Without this config  - we do not have any way to omit particular tables from profiling. would that be okay ?

Reply:
We omit tables later at profiling level where we apply the profile filter, don't we?

Reply:
I agree the profile_pattern is confusing and it would be good to be able filter early on the tables